spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/otlp/OtlpProperties.java

```diff
@@ -36,6 +39,13 @@ public class OtlpProperties extends StepRegistryProperties {
 	 */
 	private String url = "http://localhost:4318/v1/metrics";
 
+	/**
+	 * {@link AggregationTemporality} of sums. It defines the way additive values are
```

Comment:
Unfortunately, we can't use javadoc tags in the descriptions of configuration properties as it doesn't work well with configuration property metadata generation. We'll have to use `AggregationTemporality` or even just `Aggregation temporality` instead.